mjpeg-view/src/main/java/com/github/niqdev/mjpeg/MjpegInputStreamNative.java

```diff
@@ -66,24 +64,20 @@ private int getEndOfSeqeunce(DataInputStream in, byte[] sequence)
             } else seqIndex = 0;
         }
 
-
         return -1;
     }
 
-    private int getStartOfSequence(DataInputStream in, byte[] sequence)
-            throws IOException {
+    private int getStartOfSequence(DataInputStream in, byte[] sequence) throws IOException {
         int end = getEndOfSeqeunce(in, sequence);
         return (end < 0) ? (-1) : (end - sequence.length);
     }
 
-    private int getEndOfSeqeunceSimplified(DataInputStream in, byte[] sequence)
-            throws IOException {
+    private int getEndOfSeqeunceSimplified(DataInputStream in, byte[] sequence) throws IOException {
```

Comment:
@niqdev 
It's out of this pull request, but I guess you need some codestyle settings here, to avoid that much non-logic changes